src/debugger/appWorker.ts

```diff
@@ -137,7 +137,9 @@ export class SandboxedAppWorker {
     private gotResponseFromDebuggerWorker(object: any): void {
         // We might need to hold the response until a script is imported. See comments on this.importScripts()
         this.pendingScriptImport.done(() =>
-            this.postReplyToApp(object));
+            this.postReplyToApp(object), reason => {
+                printDebuggingFatalError(`Unexpected internal error while processing a message from the RN App.`, reason);
```

Comment:
Use `"` quotes rather than interpolating backticks if there is nothing to interpolate